exporters/common/build.gradle.kts

```diff
@@ -14,6 +14,7 @@ dependencies {
   api(project(":sdk-extensions:autoconfigure-spi"))
 
   compileOnly(project(":sdk:common"))
+  compileOnly(project(":exporters:common:compile-stub"))
```

Comment:
Wondering if you can explain why a separate module is needed for this, even tho its not published.

Reply:
When I try to compile a class that uses `Unsafe` with javac from jdk 17 targeting jdk 8 it fails with
```
javac --release 8 Test.java
Test.java:3: error: package sun.misc does not exist
		System.err.println(sun.misc.Unsafe.class.getName());
```
curiously when targeting jdk 11 it gets a warning instead
```
javac --release 11 Test.java
Test.java:3: warning: Unsafe is internal proprietary API and may be removed in a future release
		System.err.println(sun.misc.Unsafe.class.getName());
```
Not using `--release 8` also results in a warning
```
javac --source 8 --target 8 Test.java
warning: [options] bootstrap class path not set in conjunction with -source 8
Test.java:3: warning: Unsafe is internal proprietary API and may be removed in a future release
		System.err.println(sun.misc.Unsafe.class.getName());
```
Creating a copy of `Unsafe` just for compiling felt like the easiest way to make this work. There is a comment in our version of `Unsafe`
```
/**
 * sun.misc.Unsafe from the JDK isn't found by the compiler, we provide out own trimmed down version
 * that we can compile against.
 */
```
I could add a similar comment here also. Is this what you were looking for?

Reply:
Yeah that would be helpful.